### PR TITLE
Create handles for Tasks + Projects

### PR DESCRIPTION
**Pull request by fm3**
### Description:
- Creates classes for handling webKnossos Tasks

### Issues:
- contributes to #541

### Todos:
 - [x] create tasks from annotations
 - [x] pagination boolean, all=False
 - [x] Updated Changelog

### CODE REVIEW

File: webknossos/webknossos/client/_generated/api/default/create.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+from typing import Any, Dict
```
Comment (jstriebel) [inline webknossos/webknossos/client/_generated/api/default/create.py:1]: I guess this got a new nickname, please rename the file accordingly

File: webknossos/webknossos/annotation/__init__.py [hunk 2]
```diff
@@ -1 +1,6 @@
 from webknossos.annotation.annotation import Annotation, open_annotation
+from webknossos.annotation.annotation_info import (
+    AnnotationInfo,
+    AnnotationState,
+    AnnotationType,
+)
```
Comment (jstriebel) [inline webknossos/webknossos/annotation/__init__.py:6]:
```suggestion
from webknossos.annotation.annotation import Annotation, open_annotation, AnnotationState, AnnotationType
from webknossos.annotation.annotation_info import AnnotationInfo
```

**Review by jstriebel — COMMENTED** [review]
Great stuff, thanks a lot :heart: 

I added some comments, but mostly local changes. Overall looks great!

PS: I guess we should also add some tests, and also test the new example in `test_examples.py`

File: webknossos/webknossos/administration/project.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        all_tasks = [Task._from_generated_response(t) for t in response]
+        if total_count > pagination_limit:
+            if fetch_all:
+                while total_count > len(all_tasks):
+                    pagination_page += 1
+                    response = task_infos_by_project_id.sync(
+                        self.project_id,
+                        limit=pagination_limit,
+                        page_number=pagination_page,
+                        include_total_count=True,
```
[the remark this answers is not in the document]
Comment (fm3) [reply webknossos/webknossos/administration/project.py:84]: fair enough, it’s a tiny bit faster without it :+1:

File: webknossos/webknossos/administration/task.py [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+        instances: int,
+        project_name: str,
+        script_id: Optional[str],
+        bounding_box: Optional[BoundingBox],
+        base_annotations: List[Annotation],
+    ) -> List["Task"]:
+        """Submits tasks in webKnossos based on existing annotations, and returns the Task objects"""
+        assert (
+            len(base_annotations) > 0
+        ), "Must supply at least one base annotation to create tasks"
```
[the remark this answers is not in the document]
Comment (fm3) [reply webknossos/webknossos/administration/task.py:69]: as discussed, a second function `create` that takes a list of task parameters should be added (follow-up issue #585)

File: webknossos/__generate_client.py [hunk 5]
```diff
@@ -64,37 +63,112 @@ def add_api_prefix_for_non_data_paths(openapi_schema: Dict) -> None:
...
+    # user_id = "5b5dd2fb1c00008230ec8174"
+    # project_id = "61f1515e0100002f01249afa"
+    # project_name = "sampleProject"
+    explorative_annotation_id = "61f91018fe0100b002553010"
+    organization_name = "sample_organization"
+    dataset_name = "l4_sample"
+    task_id = "61f90e61fe0100a00255300a"
+    user_id = "61f90b45fc01000402ff5b4d"
+    project_id = "61f90b4afc01006704ff5b51"
+    project_name = "sampleProject"
```
[the remark this answers is not in the document]
Comment (fm3) [reply webknossos/__generate_client.py]: added a comment :)

File: webknossos/webknossos/client/_generated/models/annotation_info_response_200_annotation_layers_item.py [hunk 6]
```diff
@@ -7,22 +7,25 @@
 
 @attr.s(auto_attribs=True)
 class AnnotationInfoResponse200AnnotationLayersItem:
-    """ """
+    """  """
 
     tracing_id: str
     typ: str
+    name: str
```
Comment (jstriebel) [inline webknossos/webknossos/client/_generated/models/annotation_info_response_200_annotation_layers_item.py:14]: Should this be here?

File: webknossos/webknossos/client/_generated/models/annotation_infos_by_task_id_response_200_item.py [hunk 7]
```diff
@@ -52,7 +52,7 @@ class AnnotationInfosByTaskIdResponse200Item:
     tracing_store: AnnotationInfosByTaskIdResponse200ItemTracingStore
     visibility: str
     settings: AnnotationInfosByTaskIdResponse200ItemSettings
-    tracing_time: int
+    tracing_time: str
```
Comment (jstriebel) [inline webknossos/webknossos/client/_generated/models/annotation_infos_by_task_id_response_200_item.py:55]: ?

File: webknossos/webknossos/client/_generated/models/annotation_infos_by_task_id_response_200_item_task.py [hunk 8]
```diff
@@ -30,7 +30,7 @@ class AnnotationInfosByTaskIdResponse200ItemTask:
     created: int
     status: AnnotationInfosByTaskIdResponse200ItemTaskStatus
     script: str
-    tracing_time: int
+    tracing_time: str
```
Comment (jstriebel) [inline webknossos/webknossos/client/_generated/models/annotation_infos_by_task_id_response_200_item_task.py:33]: ?

**Review by jstriebel — APPROVED** [review]
Awesome stuff! See my few comments about the new client response changes

**Pull request merged**